README.md

```diff
@@ -34,7 +34,7 @@ Benchee.run(%{
     &Benchee.Formatters.HTML.output/1,
     &Benchee.Formatters.Console.output/1
   ],
-  formatter_options: [html: [file: "samples_output/my.html"]],
+  formatter_options: [html: [file: "samples_output/my.html", unit_scaling: :best]],
```

Comment:
i wasn't sure if you want it now on in the future as a top level option so I've put this it as a formatter specific argument - which i think can stay as it in case someone wants to specify units for each formatter (but that's your call).

Reply:
nah I think supporting both is too over specific, if someone will come and want differnt unit_scaling for different formatters they can come and complain but I doubt it'll happen :)